src/algebra/lie/basic.lean

```diff
@@ -840,7 +849,34 @@ instance : add_comm_monoid (lie_submodule R L M) :=
   add_zero  := λ _, sup_bot_eq,
   add_comm  := λ _ _, sup_comm, }
 
-@[simp] lemma add_eq_sup (N N' : lie_submodule R L M) : N + N' = N ⊔ N' := rfl
+@[simp] lemma add_eq_sup : N + N' = N ⊔ N' := rfl
+
+lemma sup_eq_sup_submodule :
```

Comment:
About my suggestion: making it `norm_cast` is definitely a good idea, changing the name I'm not totally convinced by.
```suggestion
@[norm_cast] lemma coe_sup :
```